src/Athens-Examples/CurveWorkshop.class.st

```diff
@@ -4,9 +4,6 @@ An example from a workshop
 Class {
```

Comment:
This class is not referenced anywhere and do not provide a way to see the example showcase. we should either just remove it, provide an open method, or move the example (#drawQuadFrom:via:to:on: canvas) into AthensDemoMorph.

This is what the CurveWorkshop shows
```Smalltalk
| surface canvas |

surface := AthensCairoSurface extent: 200@200.
canvas := surface newCanvas.

CurveWorkshop new drawOn: canvas.

surface asForm asMorph openInWindow
```